SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityScheduler.java

```diff
@@ -90,9 +92,20 @@
 
   private final SingularityMailer mailer;
 
+  private final SingularityAbort abort;
+  private final AtomicInteger numStartupRequests = new AtomicInteger(0);
```

Comment:
This is referring to tasks this time around, not requests (now that I've got my head on straight and am not leading you astray in this PR... 😉  )